src/components/Icon/Icon.component.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {createElement} from 'react';
+
+import {IconProps} from './Icon';
```

Comment:
why are we keeping file names with a capital letter?

Reply:
In Frontend, we usually keep `Components` files and folders name's with the first letter as capital

The reason for using the first capital letter is that when you search for a file by name in the project you can always recognize components just by file name `header` vs `Header`, you always know that the second one is a component and the first one is either container or page.